src/main/resources/external-user-mgmt-api.yaml

```diff
@@ -207,6 +207,42 @@ paths:
           description: Bearer token is expired, or it doesn't have the 'manage-user' scope, or the target user's roles are not included in the current user's assignable roles.
         400:
           description: Required parameters are missing or invalid.
+        404:
```

Comment:
Not sure why these bits are showing up here - presumably they haven't changed

Reply:
The 2 sections got a mixed mixed up. This has been fixed